test/unit/lib/GitHub.test.js

```diff
@@ -100,25 +100,58 @@ describe('GitHub interface', () => {
 
       const githubInstance = await new GitHub(req.params)
 
+      let exception = 'thrown'
```

Comment:
These are from the other review, they will disapear if that PR is merged (should happen first)

Reply:
I'd prefer not to merge #327 first since it does not include the fixes for failing test cases. I would prefer not to merge anything into dev that causes the build to fail.

Good catch on these exceptions. I think I would prefer the `expect.assertions(<numExceptions>)` approach used elsewhere and mentioned in #327 as it's more concise. Ideally we would be doing [something like this](http://eloquentcode.com/expect-a-function-to-throw-an-exception-in-jest), but then we couldn't test the exception body properly.

Reply:
@alexwaibel where is `expect.assertions` mentioned in 327?

Reply:
Personally I find the `expect.assertions` approach worse because it requires counting the number of expected assertions, and it doesn't give you use much useful info when it fails: it just tells you that something when wrong, but not _where_ in the code (small tests mitigates this), but sure I'll make this change.

Reply:
Sorry, it's actually mentioned in #329.

Reply:
@alexwaibel - I wasn't able to get this approach to work. I don't know how to count the number of expected assertions. For example, I would expect the line `expect(scope.isDone()).toBe(true)` to be one assertion, but it seems to count as 5.

I will withdraw this pull request.